src/bidiServer/mapperServer.ts

```diff
@@ -95,31 +102,52 @@ export class MapperServer {
     });
   }
 
-  static async #sendBidiMessage(
-    bidiMessageJson: string,
-    mapperCdpClient: CdpClient
-  ): Promise<void> {
-    await mapperCdpClient.sendCommand('Runtime.evaluate', {
+  async #sendBidiMessage(bidiMessageJson: string): Promise<void> {
+    await this.#mapperCdpClient.sendCommand('Runtime.evaluate', {
       expression: `onBidiMessage(${JSON.stringify(bidiMessageJson)})`,
     });
   }
 
   #onBidiMessage(bidiMessage: string): void {
-    // Messages in MAPPER_CHANNEL are for debug output and should not be forwarded to users.
-    try {
-      const message = JSON.parse(bidiMessage);
-      if (message.channel === MAPPER_DEBUG_CHANNEL) {
-        cdpLog(bidiMessage);
-        return;
-      }
-    } catch {}
     for (const handler of this.#handlers) handler(bidiMessage);
   }
 
   #onBindingCalled = (params: Protocol.Runtime.BindingCalledEvent) => {
     if (params.name === 'sendBidiResponse') {
       this.#onBidiMessage(params.payload);
     }
+    if (params.name === 'sendDebugMessage') {
+      this.#onDebugMessage(params.payload);
+    }
+  };
+
+  #onDebugMessage = (debugMessageStr: string) => {
+    try {
+      const debugMessage = JSON.parse(debugMessageStr) as {
+        logType: string;
+        messages: any[];
```

Comment:
unless the compilation fails, prefer `unknown[]` consistently for log messages

https://google.github.io/styleguide/tsguide.html#any

Reply:
done